test/e2e/_helpers.js

```diff
@@ -0,0 +1,230 @@
+import { find, isArray, isNil } from 'lodash'
+import { format } from 'date-fns'
+import { join } from 'path'
+import { homedir } from 'os'
+import { ClientFunction, t } from 'testcafe'
+import faker from 'faker'
+import glob from 'glob'
+
+import personService from '../../common/services/person'
+
+export const scrollToTop = ClientFunction(() => {
+  window.scrollTo(0, 0)
+})
+
+export function generatePerson(overrides = {}) {
+  const firstNames = faker.name.firstName()
+  const lastName = faker.name.lastName()
+  return {
+    lastName,
+    firstNames,
+    fullname: `${lastName}, ${firstNames}`.toUpperCase(),
+    policeNationalComputer: faker
```

Comment:
I take it we don't need the other identifier_type props ( `criminal_records_office'`, `niche_reference` or `athena_reference`?)

Reply:
That's a good thought. 

We don't capture those anywhere yet, but I think if a record has them we will display them back in the personal details section so it might be worth adding them to this so we can test they are displayed correctly when present.